neural_speed/__init__.py

```diff
@@ -73,67 +73,70 @@ def get_model_type(model_config):
             model_type = "chatglm2"
         return model_type
 
-    def init(self, model_name, not_quant=False, use_cache=False, use_gptq=False, use_awq=False,
-            weight_dtype="int4", alg="sym", group_size=32,
-            scale_dtype="fp32", compute_dtype="int8", use_ggml=False):
+    def init(self, model_name, use_quant=True, use_gptq=False, **quant_kwargs):
```

Comment:
why remove the default value of init args ?

Reply:
There was no default value on ITREX back then.

Reply:
I will ask zhenwei how to do it.

Reply:
done